include/oneapi/dpl/internal/async_impl/glue_async_impl.h

```diff
@@ -16,19 +16,23 @@
 #ifndef _ONEDPL_GLUE_ASYNC_IMPL_H
 #define _ONEDPL_GLUE_ASYNC_IMPL_H
 
-#include "../async_extension_defs.h"
+#ifdef _ONEDPL_BACKEND_SYCL
+#    include "../async_extension_defs.h"
 
-#if _ONEDPL_HETERO_BACKEND
-#    include "async_impl_hetero.h"
-#endif
+#    if _ONEDPL_HETERO_BACKEND
+#        include "async_impl_hetero.h"
+#    endif
+
+#    include "async_impl.h"
+#endif // _ONEDPL_BACKEND_SYCL
 
 namespace oneapi
 {
 namespace dpl
 {
 namespace experimental
 {
-
+#ifdef _ONEDPL_BACKEND_SYCL
```

Comment:
I think we should not to wrap the functions below in `_ONEDPL_BACKEND_SYCL` macro, because the functions are a kind of "general" for host and hetero backends.